aries-site/src/layouts/content/Example/Example.js

```diff
@@ -22,11 +22,13 @@ export const Example = ({
   background,
   children,
   code, // github code link used to display code inline
+  componentName,
```

Comment:
Can we add these props to the propTypes definition at the bottom of the file?

Are you not receiving an eslint warning saying they are not defined?